astropy_helpers/commands/build_py.py

```diff
@@ -21,19 +21,6 @@ def finalize_options(self):
 
         SetuptoolsBuildPy.finalize_options(self)
 
-    def run_2to3(self, files, doctests=False):
-        # Filter the files to exclude things that shouldn't be 2to3'd
-        skip_2to3 = self.distribution.skip_2to3
-        filtered_files = []
-        for filename in files:
-            for package in skip_2to3:
-                if filename[len(self.build_lib) + 1:].startswith(package):
-                    break
-            else:
-                filtered_files.append(filename)
-
-        SetuptoolsBuildPy.run_2to3(self, filtered_files, doctests)
-
```

Comment:
I think the whole file can be removed - the splitting of the build directories was done because of Python 2/3 but that actually causes problems elsewhere. Since this isn't needed anymore, I suggest removing the whole custom build command.